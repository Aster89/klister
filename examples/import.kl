#lang kernel

[import (only "n-ary-app.kl" #%app)]
[import (only "n-ary-app.kl" const)]
[import (only "n-ary-app.kl" lambda)]

[example (const [quote one] [quote two])]
[example ([lambda (x y z) y] [quote en] [quote to] [quote tre])]
