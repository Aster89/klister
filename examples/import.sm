#lang kernel

[import "n-ary-app.sm" #%app]
[import "n-ary-app.sm" const]

[example (const [quote one] [quote two])]
