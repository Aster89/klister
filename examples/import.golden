#[import.kl:7.24-7.27]<one> : Syntax
#[import.kl:8.48-8.50]<to> : Syntax
